.github/workflows/publish.yml

```diff
@@ -10,16 +10,12 @@ on:
   # Triggers the workflow on push or pull request events but only for the main branch
   push:
     branches: [ main ]
-  pull_request:
-    branches: [ main ]
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
-jobs:
```

Comment:
hi @pmanko ,did you just  remove the `jobs:`  accidentally ??
wont the ci build fail ??

Reply:
hi @pmanko ,
i guess you removed the **jobs:**  accidentally  . you can just fix this and we merge this PR